src/lib/core/index.js

```diff
@@ -1,12 +1,49 @@
 import React from "react";
 import { withMutations } from "../mutate";
 import Frame from "../frame";
+import { connect } from "react-redux";
 import { Themed } from "../theme";
+import { setToast } from "../../redux/actions";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.min.css";
 
-const Core = () => (
-  <Themed>
-    <Frame />
-  </Themed>
-);
+class Core extends React.Component {
+  constructor(props) {
+    super(props);
+    this.props.dispatch(setToast("do you like cats?", "info"));
+  }
 
-export default withMutations(Core);
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.toast) {
```

Comment:
Okay last thing I swear 😅

In Javascript (and other callback-heavy languages), prefer the [guard](https://thatthinginswift.com/guard-statement-swift/) style if statements in order to avoid indent-hadouken (or the "tower of doom"): 

![extremely indented code](http://i.imgur.com/BtjZedW.jpg)

You can implement this pattern in javascript like this:
``` js
if (!nextProps.toast) { return; } 

// This stuff will only happen if toast exists
```

Reply:
I actually was thinking about doing this, but I was concerned about the situation where other properties will be passed into `componentWillReceiveProps`. Using a return statement will work for this single instance, but disables all other future properties from being checked. Thoughts?